app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -28,7 +28,14 @@ function initializeCommentDropdown() {
     }
   }
 
-  function copyText() {
+  function copyPermalink(event) {
+    event.preventDefault();
+    const permalink = event.target.href;
+    
+    Runtime.copyToClipboard(permalink);
```

Comment:
We can take advantage of the Snackbar to give users visual feedback here (catch to avoid unexpected problem if the Snackbar happens to not be available).

```javascript
Runtime.copyToClipboard(permalink).then(() => {
   addSnackbarItem({ message: 'Copied to clipbaord' });
}).catch((e) => { console.log(e); });
```

Reply:
Sounds good :D I did have to import `addSnackbarItem` though. :+1: But this should be resolved :slightly_smiling_face: